### PR TITLE
SMSI-751: Add the confidence field to VPN response

### PR DESCRIPTION
**Pull request by soniabhishek**

### CODE REVIEW

**Comment by soniabhishek** [comment]
Hi team @JuroUhlar @ilfa can any one of you can review or assign this PR for someone by today EOD ? We are looking for a release tomorrow (so this change is important to be in the rollout state)

File: schemas/components/schemas/VpnResult.yaml [hunk 1]
```diff
@@ -46,6 +46,12 @@ properties:
       - publicVPN
       - auxiliaryMobile
       - osMismatch
+  confidence:
+    type: string
+    description: >-
```
Comment (ilfa) [inline schemas/components/schemas/VpnResult.yaml:51]: Looks like [enum](https://swagger.io/docs/specification/data-models/enums/), please add if you agree

**Review by ilfa — CHANGES_REQUESTED** [review]

File: schemas/components/schemas/VpnResult.yaml [hunk 2]
```diff
@@ -46,6 +46,12 @@ properties:
       - publicVPN
       - auxiliaryMobile
       - osMismatch
+  confidence:
+    type: string
```
Comment (JuroUhlar) [inline schemas/components/schemas/VpnResult.yaml:50]: Feels like a good place to use enum? 
```suggestion
    type: string
    enum: [low, medium, high]
```

File: schemas/components/schemas/VpnResult.yaml [hunk 3]
```diff
@@ -46,6 +46,12 @@ properties:
...
+    description: >-
+      provides the confidence of the detected vpn based on reliability of source and algorithm
+      we will be categorising the response into low/medium/high (which are the 3 supported values for the response type)
```
Comment (JuroUhlar) [inline schemas/components/schemas/VpnResult.yaml:53]:
```suggestion
    description: >-
      A confidence rating for the VPN detection result — "low", "medium", or "high". Depends on the combination of results returned from all VPN detection methods.
```

File: schemas/components/schemas/VpnResult.yaml [hunk 4]
```diff
@@ -46,6 +46,12 @@ properties:
...
+    example: low
 required:
```
Comment (JuroUhlar) [inline schemas/components/schemas/VpnResult.yaml:55]: Is `confidence` returned for both positive and negative results? Then is should be in the listed in `required`, right? 
If not, the description should specify that it is returned only for positive results.

Comment (soniabhishek) [reply schemas/components/schemas/VpnResult.yaml:55]: Yes, will always be present 
I have added this to the required column

**Review by JuroUhlar — DISMISSED** [review]

**Comment by JuroUhlar** [comment]
@soniabhishek Thanks for the changes, looks good to me now, but the pipeline is failing because VPN confidence is not yet released. Once it is, you can rerun the pipeline and merge the PR. 
cc @ilfa

**Comment by soniabhishek** [comment]
Sure @JuroUhlar, thanks for quick response (I will be releasing the changes today and will rerun the CI to confirm if everything is working as expected)

File: .changeset/orange-taxis-fetch.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+'fingerprint-pro-server-api-openapi': minor
+---
+
+**visitors**: Add the confidence field to VPN response
```
Comment (ilfa) [inline .changeset/orange-taxis-fetch.md:5]:
```suggestion
**visitors**: Add the confidence field to the VPN Detection Smart Signal
```

**Review by ilfa — APPROVED** [review]

**Comment by soniabhishek** [comment]
Changes have been deployed and there after CI turned green (on rerun)
I will move ahead with the merge of this PR
Thanks everyone

**Pull request merged**